lib/components/viewers/route-details.js

```diff
@@ -145,20 +144,15 @@ class RouteDetails extends Component {
               </>
             )}
             {url && (
-              <a
-                href={url}
-                rel="noreferrer"
+              <LinkOpensNewWindow
+                contents={
+                  <FormattedMessage id="components.RouteDetails.moreDetails" />
```

Comment:
(From WAVE a11y tool) "More Details" can be ambiguous as per [WCAG](https://www.w3.org/TR/WCAG21/#link-purpose-in-context)
![image](https://user-images.githubusercontent.com/62163307/232609746-dc5ba298-da3a-444f-a966-b6bc266a85a5.png)

Maybe we can change it to something like "View Agency Website"?

Reply:
@miles-grant-ibigroup @AdrianaCeric spoke to the external a11y auditors today and they seemed to think this was alright for sighted users, since it is the only link of its kind on this page, but that we may want to include an invisible span with additional details for AT. 

I think I'd personally prefer keeping "more details" with the external site icon & invisible span as opposed to changing the link text, but I wanted to get both of your thoughts!

Reply:
I think `InvisibleA11yLabel` could work here too! I'm not too biased towards either